core/commands/storage/stats_cmd.go

```diff
@@ -79,6 +79,14 @@ func SyncStats(ctx context.Context, cfg *config.Config, node *core.IpfsNode, env
 		Online:               cfg.Experimental.StorageHostEnabled,
 		Uptime:               sr.Uptime,
 		Score:                sr.Score,
+		UptimeScore:          sr.UptimeScore,
+		AgeScore:             sr.AgeScore,
+		VersionScore:         sr.VersionScore,
+		SpeedScore:           sr.SpeedScore,
+		UptimeWeight:         sr.UptimeWeight,
+		AgeWeight:            sr.AgeWeight,
+		VersionWeight:        sr.VersionWeight,
+		SpeedWeight:          sr.SpeedWeight,
```

Comment:
actual stat command is not showing this (yet)

need to implement it in `btfs storage contracts stat host`

Reply:
ok, nvm, it is auto unmarshalled.